### PR TITLE
Adapt def _assign_imports_to_globals

### PR DESCRIPTION
**Pull request by triper1022**
Hi~

I found we often make multiple imports in one line like below
from keras.layers import LSTM, Dropout, Dense.

Therefore, I rewrote the func., def _assign_imports_to_globals() to make this work.
If you are free, please check it out.

### CODE REVIEW

**Comment by FlorianWetschoreck** [comment]
Hey, thank you for your PR!

Please describe the use case a little bit more in detail.
E.g. do you add the "from keras.layers import LSTM, Dropout, Dense" to your personal pyforest imports?
What is the behavior of pyforest before your PR and after?
Also, we would need some tests for the PR.

**Comment by triper1022** [comment]
Hi~ Thanks for your reply.

E.g. do you add the "from keras.layers import LSTM, Dropout, Dense" to your personal pyforest imports?
Yes, I did. I knew that in original project, 
the normal accepted formats are "import matplotlib.pyplot as plt" or 
"from sklearn.ensemble import GradientBoostingRegressor"

However, 
I know that we often would like to make the lines shorter, 
we usually write imports in one line with comma 
if they are in the same module.
We don't write 
"from keras.layers import LSTM", 
"from keras.layers import Dropout", 
"from keras.layers import Dense" repeatedly.
The similar situation also happens when using Python built-in modules.
E.g., "import sys, os, re"

My PR is to split the string, 
"from keras.layers import LSTM, Dropout, Dense",
and create 3 LazyImport instances as the original project, 
LSTM = LazyImport("from keras.layers import LSTM"),
Dropout = LazyImport("from keras.layers import Dropout"),
Dense = LazyImport("from keras.layers import Dense"), 
but users still can add their common use imports 
in ~/.pyforest/user_imports.py, 
like "from keras.layers import LSTM, Dropout, Dense" in one line.

Besides, I created 2 small personal features in [my forked repo](https://github.com/triper1022/pyforest).
If you have interest, welcome to glance at them.
I wrote short description in readme file.

I think this project is useful for 
the users who often change their workplaces
in different local machines, cloud IDEs, or projects.
Thank you very much!

**Comment by FlorianWetschoreck** [comment]
Thank you for your reply. Please also describe the behavior/spec of pyforest before and after your PR.

**Comment by tkrabel** [comment]
Hi @triper1022,

So you mentioned that your PR allows writing import statements like `from keras.layers import LSTM, Dropout, Dense` to your personal pyforest imports.

Now, I'm wondering: If you open a fresh Notebook (with no imports written yet) and start using `keras.layers.Dropout` for example (but not LSTM or Dense), what import statement will be added to the top of the Notebook?

Will it be 
- `from keras.layers import LSTM, Dropout, Dense`
- `from keras.layers import Dropout`
- something else?

Can you describe any other behavioural changes of pyforest coming with your PR?

File: src/pyforest/user_specific_imports.py [hunk 1]
```diff
@@ -60,9 +60,25 @@ def _get_imports_from_user_settings(user_imports_path) -> list:
...
-
-    for symbol, import_statement in zip(symbols, import_statements):
+    symbols = []; new_import_statements = []
+    
+    for import_statement in import_statements:
+        def process(statement):
+            symbols.append(statement.split()[-1])
+            new_import_statements.append(statement)
+
+        if "," not in import_statement:
```
Comment (triper1022) [inline src/pyforest/user_specific_imports.py:70]: Thanks for reply!

I didn't change any the original behaviors.

I only put line 70. to judge whether the import statement include multiple imports.

if "," not in import_statement,
"from keras.layers import LSTM" still behaves as the same as original code.

if "," is in import_statement, "from keras.layers import LSTM, Dropout, Dense",
it will become
"from keras.layers import LSTM"
"from keras.layers import Dropout"
"from keras.layers import Dense".
Then, the following behaviors are still the same as the original

File: src/pyforest/user_specific_imports.py [hunk 2]
```diff
@@ -60,9 +60,25 @@ def _get_imports_from_user_settings(user_imports_path) -> list:
...
+            process(import_statement)
+        else:
```
Comment (triper1022) [inline src/pyforest/user_specific_imports.py:72]: Thanks for reply!

My PR doesn't affect any original behaviors.

PR uses new line 70. to judge whether the import statement includes multiple imports at first.

if "," not in import_statement,
"from keras.layers import LSTM" still behaves as original code
like original line 63, which is similar with new line 66~68.

if "," is in import_statement, "from keras.layers import LSTM, Dropout, Dense",
it will become
"from keras.layers import LSTM"
"from keras.layers import Dropout"
"from keras.layers import Dense" with new line 73~78.

Then, the following behaviors are still the same as the original line 63.

In the other words,
I extend the original line 63 into new line 63~78 for the judgement and split.

File: src/pyforest/user_specific_imports.py [hunk 3]
```diff
@@ -60,9 +60,24 @@ def _get_imports_from_user_settings(user_imports_path) -> list:
...
```
Comment (triper1022) [inline src/pyforest/user_specific_imports.py:72]: Thanks for reply!

My PR doesn't affect any original behaviors.

At first,
PR uses new line 70. to judge whether the import statement includes multiple imports.

if "," not in import_statement,
"from keras.layers import LSTM" still behaves as original code
of original line 63, which is similar with new line 63~68.

if "," is in import_statement, 
like "from keras.layers import LSTM, Dropout, Dense",
it will become
"from keras.layers import LSTM"
"from keras.layers import Dropout"
"from keras.layers import Dense" with new line 73~78.

Then, the following behaviors are still the same as the original line 63, 65, 66.
